tls/tls_srv.c

```diff
@@ -883,6 +894,11 @@ ttls_parse_client_hello(TlsCtx *tls, unsigned char *buf, size_t len,
 		}
 		io->hslen -= 2;
 		tls->hs->cs_cur_len += 2;
+
+		/* ja5t must be initialized with zeros */
+		tls->sess.ja5t.cipher_suite_hash *= TTLS_JA5_HASH_CALC_PRIME;
+		tls->sess.ja5t.cipher_suite_hash += cs;;
```

Comment:
```suggestion
		tls->sess.ja5t.cipher_suite_hash += cs;
```

These 2 lines implement our hashing, which could change -what about to make a macro to compute a hash for any field? Like
```c
#define TTLS_COMPUTE_JA5_ACCHASH(hash, field)                                   \              
do {                                                                            \              
        (hash) *= TTLS_JA5_HASH_CALC_PRIME;                                     \              
        (hash) += (field);                                                      \              
} while (0)                   
```

Reply:
done